JapaneseFilter/sections/specific.txt

```diff
@@ -35,6 +35,9 @@ pokemon-gsc.g-takumi.com,1rk.net,e-typing.ne.jp,zukai-kikenbutu.com,gingerweb.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+5chan.jp##.linklist > li > a:contains([PR])
+5chan.jp##.socialandbanner
```

Comment:
Maybe replace with `5chan.jp##.toppageadvheader` and add `5chan.jp##.socialicons` to Social. @zloyden What do you think? Test on PC for this element.

Reply:
Actually, I can't open the website with VPN. Can you show why your changes are needed?

Reply:
> but there's actually no SNS staff

Thank you. Just leave it as it is.

Reply:
> Screenshot (NSFW)

For the future, just leave a link without a preview for such screenshots - I have edit your message.

Reply:
Fixed.